hydra-tui/src/Hydra/TUI/Drawing.hs

```diff
@@ -8,7 +8,7 @@ module Hydra.TUI.Drawing where
 import Hydra.Prelude hiding (Down, State, padLeft)
 
 import Brick
-import Hydra.Cardano.Api hiding (Active)
+import Hydra.Cardano.Api hiding (Active, blue, green, red, (<+>))
```

Comment:
Must: hide weird exports in the corruption layer (= in the `Hydra.Cardano.API` module)